pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,95 @@ def set_view(self, viewType):
+            'korea': 5,
+            'hijri': 6,
+            'thai': 7,
+            'hebwer': 8,
+            'gregorian_french_me': 9,
+            'gregorian_arabic': 10,
+            'gregorian_english_xlit': 11,
+            'gregorian_french_xlit': 12,
+            'umalqura': 23
+        }
+        if ID in dict_types:
+            self.send_message(win32defines.MCM_SETCALID, dict_types[ID], 0)
+        else:
+            raise RuntimeError('You typed incorrect type of Calendar')
+
+    # ----------------------------------------------------------------
+    def get_id(self):
+        """Get type of calendar"""
+        return self.send_message(win32defines.MCM_GETCALID, 0, 0)
+
+    # ----------------------------------------------------------------
+    def set_color(self, place_of_color, red, green, blue):
+        """
+        Set some color in some place of calendar.
+        Receive four parameters:
+        - First may takes few variants below:
+        'background', 'month_background', 'text', 'title_background',
+        'title_text', 'trailing_text' ;
+        - All other parameters should be integer from 0 to 255.
+        """
+        place_in_calendar = {
+            'background': 'MCSC_BACKGROUND',
+            'month_background': 'MCSC_MONTHBK',
+            'text': 'MCSC_TEXT',
+            'title_background': 'MCSC_TITLEBK',
+            'title_text': 'MCSC_TITLETEXT',
+            'trailing_text': 'MCSC_TRAILINGTEXT'
+        }
+        if red < 0 and red < 255:
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        if green < 0 and green < 255:
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        if blue < 0 and blue < 255:
+            raise RuntimeError('Incorrect range of color, must be from 0 to 255')
+        color = (red, green, blue)
+        if place_of_color in place_in_calendar:
+            result = self.send_message(win32defines.MCM_SETCOLOR, place_of_color, color)
+        else:
+            raise RuntimeError('Incorrect place ID for color')
+        if result == -1:
+            raise RuntimeError('Incorrect color')
+
+    # ----------------------------------------------------------------
+    def get_color(self, place_of_color):
+        """
+        Return COLORREF in place_of_color which want.
+        Receive only one parameter, which takes variants below:
+        'background', 'month_background', 'text', 'title_background', 'title_text', 'trailing_text'
+        """
+
+        place_in_calendar = {
+            'background': 'MCSC_BACKGROUND',
+            'month_background': 'MCSC_MONTHBK',
+            'text': 'MCSC_TEXT',
+            'title_background': 'MCSC_TITLEBK',
+            'title_text': 'MCSC_TITLETEXT',
+            'trailing_text': 'MCSC_TRAILINGTEXT'
+        }
+        if place_of_color in place_in_calendar:
+            return self.send_message(win32defines.MCM_GETCOLOR, place_of_color, 0)

```

Comment:
:) It's very easy to google these constants values. See [here](http://perl-win32-gui.sourceforge.net/cgi-bin/docs.cgi?doc=monthcal) for example.